apps/SageAccountWeb/src/style/theme.ts

```diff
@@ -56,48 +59,6 @@ export const themeOptions = mergeTheme({
         },
       },
     },
-    MuiDialog: {
```

Comment:
Dialog styles were removed or moved to synapse-react-client. No reason all apps can't use this style, what's in SageAccountWeb is not substantially different from the default.